private/extensions/maven.bzl

```diff
@@ -412,6 +412,7 @@ def _maven_impl(mctx):
                 # created from the maven_install.json file in the coursier_fetch
                 # invocation after this.
                 name = "unpinned_" + name if repo.get("lock_file") else name,
+		pinned_repo_name = name if repo.get("lock_file") else None,
```

Comment:
could you fix up the indent here?

Reply:
Whoops, didn't see that a tab got in here somehow. Good catch. Fixed.